lib/web_of_science/query_author.rb

```diff
@@ -34,10 +34,38 @@ def uids
       # @return [Hash]
       def author_query
         params = queries.params_for_fields(empty_fields)
-        params[:queryParameters][:userQuery] = "AU=(#{names}) AND AD=(#{institution})"
+        update_time_span(params)
+        update_user_query(params)
         params
       end
 
+      # Use options to limit the symbolic time span for harvesting publications; this limit applies
+      # to the dates publications are added or updated in WOS collections, not publication dates. To
+      # quote the API documentation:
+      #
+      #     The symbolicTimeSpan element defines a range of load dates. The load date is the date when a record
+      #     was added to a database. If symbolicTimeSpan is specified, the timeSpan parameter must be omitted.
+      #     If timeSpan and symbolicTimeSpan are both omitted, then the maximum publication date time span
+      #     will be inferred from the editions data.
+      #
+      #     The valid values are strings: '1week', '2week', '4week' (prior to today)
+      #
+      # @param [Hash] params from queries.params_for_fields
+      # @return [void]
+      def update_time_span(params)
+        return if options[:update].blank?
+        params[:queryParameters].delete(:timeSpan)
```

Comment:
may want to put an inline comment here indicating why this is done for the benefit of future developers (i.e. WoS documentation states you can't have both timeSpan and symbolicTimeSpan)

Reply:
That documentation is all there in the method docs already.

Reply:
ok, missed it